conf/app.ini

```diff
@@ -20,6 +20,26 @@ MAX_CREATION_LIMIT = -1
 ; Patch test queue length, make it as large as possible
 PULL_REQUEST_QUEUE_LENGTH = 10000
 
+[editor]
+; List of file extensions that should have line wraps in the CodeMirror editor
+; Separate extensions with a comma. To line wrap files w/o extension, just put a comma
+LINE_WRAP_EXTENSIONS = .txt,.md,.markdown,.mdown,.mkd,,
```

Comment:
OK, good point...I thought the two commas made it more clear, but right, doesn't need two.